docs/types/linter-message-v2.md

```diff
@@ -0,0 +1,56 @@
+# Linter Message v2
+
+This document describes the type of Linter Message v2. It's supported in [`Indie Linter v2`](indie-linter-v2.md) and [`Standard Linter v2`](standard-linter-v2.md).
+
+## Type
+
+```js
+type Message = {
+  // Automatically added, not guaranteed
```

Comment:
This should also explicitly note that Linter providers are _not_ to provide these values themselves.